tools/ci_build/github/azure-pipelines/templates/stages/mac-ios-packaging-build-stage.yml

```
@@ -117,16 +115,66 @@ stages:
         useXcpretty: false  # xcpretty can hide useful error output so we will disable it
       displayName: 'Build App Center iPhone arm64 tests'
 
+    - script: |
+        zip -r --symlinks $(Build.ArtifactStagingDirectory)/package_tests.zip ios_package_testUITests-Runner.app
+      workingDirectory: '$(Build.BinariesDirectory)/app_center_test/apple_package_test/DerivedData/Build/Products/Debug-iphoneos'
+      displayName: "Create .zip file of the tests"
+
+    - script: |
+        python $(Build.SourcesDirectory)/onnxruntime/test/platform/apple/generate_ipa_export_options_plist.py --dest_file "exportOptions.plist" --apple_team_id $(APPLE_TEAM_ID) --provisioning_profile_uuid $(APPLE_PROV_PROFILE_UUID)
```

nit: this line is kind of long
```suggestion
        python $(Build.SourcesDirectory)/onnxruntime/test/platform/apple/generate_ipa_export_options_plist.py \
          --dest_file "exportOptions.plist" \
          --apple_team_id $(APPLE_TEAM_ID) \
          --provisioning_profile_uuid $(APPLE_PROV_PROFILE_UUID)
```